src/components/ButtonLink/ButtonLink.stories.tsx

```diff
@@ -24,6 +24,7 @@ export const Single = () => (
             alert('Clicked!');
         }}
         pillar="lifestyle"
+        linkName="ophanLink"
```

Comment:
pedantic, but lowercase hyphenate `ophanLink` if that's the casing we would prefer generally?

Reply:
Yeah, good point, these are actually placeholders so might just make them empty strings for `stories`